common/src/main/java/com/wynntils/wynn/handleditems/items/game/AmplifierItem.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wynn.handleditems.items.game;
+
+import com.wynntils.wynn.handleditems.properties.NumberedTierItemProperty;
+
+public class AmplifierItem extends GameItem implements NumberedTierItemProperty {
```

Comment:
All these item classes look like they could be a record. Can records do inheritance?

Reply:
No, records cannot do inheritance :-(

Otherwise I agree that they look very record-y. Unfortunately records is only half of the story about what Java is missing. *Some* cases where classes are simple data storage units fits as records, but not all. Apparently this was a concious (but contended!) design decision. I'm not sure I fully understand it, nor agree with it. But that's where we are. So the WynnItems are "old-school records"...